gravitee-apim-console-webui/src/management/api/endpoints-v4/backend-services/endpoint-group/general/api-endpoint-group-general.component.html

```diff
@@ -0,0 +1,34 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<form *ngIf="!!generalForm" [formGroup]="generalForm" autocomplete="off" gioFormFocusInvalid>
+  <mat-card>
+    <!-- Name -->
+    <div class="card__endpoint-group-name">
+      <div class="card__endpoint-group-name__label">Name</div>
```

Comment:
Labels could be taken from a file - this would allow internationalisation in future but maybe something for future tech debts

Reply:
Actually, it's not planned to translate the console so you can keep it like this.

_Note: The developer portal is designed to be customized and the labels are all read from language files._

Reply:
Thanks Florent!